src/content/docs/es/reference/errors/session-config-missing-error.mdx

```diff
@@ -9,5 +9,5 @@ githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/
 Se lanza cuando el almacenamiento de sesión está habilitado pero no se ha configurado.
 
 **Ver también:**
--  [experimental.session](/es/reference/experimental-flags/sessions/)
+-  [API de Sesión](/es/guides/sessions/)
```

Comment:
Since the English links are just "Sessions", we can use the more proper translation here.
```suggestion
-  [Sesiones](/es/guides/sessions/)
```